### ISSUE TITLE
Route Inflections

### ISSUE THREAD

**Issue by pusewicz** [enriclluelles/route_translator#121]
Is it possible to translate the `new` part of the path for different resources?

In Spanish `new` would translate to `nuevo` or `nueva` depending on the gender of the resource.

**Comment by tagliala** [comment]
Hi,

it should not be possible at the moment, I don't think this gem supports inflections

**Comment by pusewicz** [comment]
Yeah, but having the ability to define the translation per resource would be enough.

**Comment by tagliala** [comment]
@pusewicz I was thinking about the same feature

Labeling as "help wanted", everyone feel free to submit a PR with proper tests.

Please do not +1 here

**Comment by sunny** [comment]
A good way to do this is to change the default `new` to something you can translate.

```
localized do
  resources :categories, path_names: { new: "new_category" }
end
```

In your translations you can now translate `new_category` to `new` in english and `nueva` in spanish.

**Comment by tagliala** [comment]
I'm going to close this as "not planned", because

1. It is possible to have this behavior and it is documented in the readme https://github.com/enriclluelles/route_translator?tab=readme-ov-file#inflections
2. It could be possible to have gender-neutral actions instead of `new` (eg: in Italian, we can have `crea` / `aggiungi` instead of `nuovo/a`
